src/code42cli/cmds/legal_hold.py

```diff
@@ -227,12 +242,12 @@ def _add_user_to_legal_hold(sdk, matter_id, username):
     sdk.legalhold.add_to_matter(user_id, matter_id)
 
 
-def _remove_user_from_legal_hold(sdk, matter_id, username):
-    _check_matter_is_accessible(sdk, matter_id)
+def _remove_user_from_legal_hold(state, matter_id, username):
```

Comment:
We had these functions take the sdk directly so that in bulk commands it wouldn't have to initialize the sdk in each thread (so as to make only 1 auth call instead of 5).

Reply:
ah, I see, we need the state to check the profile. I suppose we could just initialize the sdk by accessing the property on the state before passing it in on the bulk commands then.

Reply:
I see, yeah I can look into updating the bulk methods